modules/guides/partials/clients.adoc

```diff
@@ -0,0 +1,22 @@
+:imagesdir: ../assets/images
+
+=== Couchbase Clients
+
+// tag::body[]
+Clients access data by connecting to a Couchbase cluster over the network.
+The most common type of client is a Couchbase SDK, which is a full programmatic API that enables applications to take the best advantage of Couchbase.
+The command line clients also provide a quick and streamlined interface for simple access and are suitable if you just want to access an item without writing any code.
+
+NOTE: The Command Line Clients are provided as part of your Couchbase Server installation in `/opt/couchbase/bin`.
+
+For further information see:
```

Comment:
Need another tag so can reference this snippet on other pages.

```suggestion
// tag::refs[]
Read the following for further information about the clients available for key-value operations:
```

Reply:
You might have missed this one — please can you update the lead-in paragraph and include it in the `refs` tagged area.